src/app/my-skills/my-skills-dialog-component-trait.ts

```diff
@@ -0,0 +1,12 @@
+import { StepDescription } from './step-description';
+
+export abstract class MySkillsDialogComponentTrait {
+
+  protected getLabelHint(description: StepDescription): string {
+    return '0 - ' + description.step0 + '\n' +
```

Comment:
Just an idea: This can be written as ES6 template string (in backticks) to get rid of the concatenation. For example:

```
return `0 - ${description.step0}
        1 - ${description.step1} ... `;
```

Reply:
Great idea! Thank you